src/Phoenix/PhoenixClient.ts

```diff
@@ -72,7 +76,7 @@ export class PhoenixClient {
     try {
       const runContainerStatusAsync = async () => {
         const response = await window.fetch(
-          `${this.getPhoenixContainerPoolingEndPoint()}/${containerData.dbAccountName}/${containerData.forwardingId}`,
+          `${this.getPhoenixControlPlaneEndpoint()}${userContext.databaseAccount.id}/${containerData.forwardingId}`,
```

Comment:
move using "${userContext.databaseAccount.id}" to getPhoenixControlPlaneEndpoint itself.